File: TransCoder_tokenized_test_set_functions/python/REARRANGE_ARRAY_ARRI_ARRJ_EVEN_ARRI.py
def rearrangeArr ( arr, n ) :
    evenPos = int ( n / 2 )
    oddPos = n - evenPos
    tempArr = np . empty ( n, dtype = object )
    for i in range ( 0, n ) :
        tempArr [ i ] = arr [ i ]
    tempArr . sort ( )
    j = oddPos - 1
    for i in range ( 0, n, 2 ) :
        arr [ i ] = tempArr [ j ]
        j = j - 1
    j = oddPos
    for i in range ( 1, n, 2 ) :
        arr [ i ] = tempArr [ j ]
        j = j + 1
    for i in range ( 0, n ) :
        print ( arr [ i ], end = ' ' )
    

File: TransCoder_tokenized_test_set_functions/java/REARRANGE_ARRAY_ARRI_ARRJ_EVEN_ARRI.java
public static void rearrangeArr ( int arr [ ], int n ) {
  int evenPos = n / 2;
  int oddPos = n - evenPos;
  int [ ] tempArr = new int [ n ];
  for ( int i = 0;
  i < n;
  i ++ ) tempArr [ i ] = arr [ i ];
  Arrays . sort ( tempArr );
  int j = oddPos - 1;
  for ( int i = 0;
  i < n;
  i += 2 ) {
    arr [ i ] = tempArr [ j ];
    j --;
  }
  j = oddPos;
  for ( int i = 1;
  i < n;
  i += 2 ) {
    arr [ i ] = tempArr [ j ];
    j ++;
  }
  for ( int i = 0;
  i < n;
  i ++ ) System . out . print ( arr [ i ] + " " );
}